### DISCUSSION TITLE
Nix package version

### DISCUSSION THREAD

**Discussion by ghost** [artempyanykh/marksman#74] [General]
Hi, I couldn't get the prebuilt libraries to work on NixOS. Any chance a nix package version will be created in the near future? It would be very much appreciated.

**Comment by artempyanykh** [comment]
Hi @FerencLukacs! Thanks for raising this.

> Hi, I couldn't get the prebuilt libraries to work on NixOS

Do you mean you tried making a nix package but something didn’t work? Or just the fact that there’s no nix package for Marksman?

**Reply by ghost** [reply]
Oops, that's a typo, I meant the prebuilt binaries. So i just downloaded the linux version and tried to run it as per the instructions in the readme, and it didn't work. I assumed it's because of some peculiarity of NixOS.

**Reply by artempyanykh** [reply]
Huh, interesting. Is there an error message if you run `marksman server -v 4`?

**Comment by artempyanykh** [comment]
@ghost would you try a nix flake https://github.com/the-mikedavis/marksman/blob/flake/flake.nix from here? If it works I'll add it to the mainline.

**Comment by pacien** [comment]
For the record, nixpkgs now has a package for marksman on the stable channel:
https://search.nixos.org/packages?query=marksman
